SETUP.md

```diff
@@ -83,13 +83,8 @@ To perform backup, BBQ needs rights to read BigQuery data from the project which
 * Grant this permission for the whole folder or organisation. It will be inherited by all of the projects underneath.
 
 ### Cloud Datastore export
-  BBQ may periodically export data from Datastore to Big Query. It's much easier to query the data in Big Query rather than Datastore. To enable export:
-
-* Execute command below which assigns the **Cloud Datastore Import Export Admin** IAM role to BBQ default service account:
```

Comment:
What about removed command? I think that BBQ default account should still have Cloud Datastore Import Export Admin role on BBQ project to be able to export data from datastore, and also have additional accesses to be able export that data outside of bbq project. 

Why we want to remove that from documentation?

Reply:
Discusses offline :)